packages/@aws-cdk/aws-cognito-identitypool/test/integ.identitypool.expected.json

```diff
@@ -0,0 +1,318 @@
+          "EmailSubject": "Verify your new account",
+          "SmsMessage": "The verification code to your new account is {####}"
+        }
+      },
+      "UpdateReplacePolicy": "Retain",
+      "DeletionPolicy": "Retain"
+    },
+    "PoolUserPoolClientForundefinedBF6BDE57": {
+      "Type": "AWS::Cognito::UserPoolClient",
+      "Properties": {
+        "UserPoolId": {
+          "Ref": "PoolD3F588B8"
+        },
+        "AllowedOAuthFlows": [
+          "implicit",
+          "code"
+        ],
+        "AllowedOAuthFlowsUserPoolClient": true,
+        "AllowedOAuthScopes": [
+          "profile",
+          "phone",
+          "email",
+          "openid",
+          "aws.cognito.signin.user.admin"
+        ],
+        "CallbackURLs": [
+          "https://example.com"
+        ],
+        "SupportedIdentityProviders": [
+          "COGNITO"
+        ]
+      }
+    },
+    "OtherPool7DA7F2F7": {
+      "Type": "AWS::Cognito::UserPool",
+      "Properties": {
+        "AccountRecoverySetting": {
+          "RecoveryMechanisms": [
+            {
+              "Name": "verified_phone_number",
+              "Priority": 1
+            },
+            {
+              "Name": "verified_email",
+              "Priority": 2
+            }
+          ]
+        },
+        "AdminCreateUserConfig": {
+          "AllowAdminCreateUserOnly": true
+        },
+        "EmailVerificationMessage": "The verification code to your new account is {####}",
+        "EmailVerificationSubject": "Verify your new account",
+        "SmsVerificationMessage": "The verification code to your new account is {####}",
+        "VerificationMessageTemplate": {
+          "DefaultEmailOption": "CONFIRM_WITH_CODE",
+          "EmailMessage": "The verification code to your new account is {####}",
+          "EmailSubject": "Verify your new account",
+          "SmsMessage": "The verification code to your new account is {####}"
+        }
+      },
+      "UpdateReplacePolicy": "Retain",
+      "DeletionPolicy": "Retain"
+    },
+    "OtherPoolUserPoolClientForundefined1B97829F": {
+      "Type": "AWS::Cognito::UserPoolClient",
+      "Properties": {
+        "UserPoolId": {
+          "Ref": "OtherPool7DA7F2F7"
+        },
+        "AllowedOAuthFlows": [
+          "implicit",
+          "code"
+        ],
+        "AllowedOAuthFlowsUserPoolClient": true,
+        "AllowedOAuthScopes": [
+          "profile",
+          "phone",
+          "email",
+          "openid",
+          "aws.cognito.signin.user.admin"
+        ],
+        "CallbackURLs": [
+          "https://example.com"
+        ],
+        "SupportedIdentityProviders": [
+          "COGNITO"
+        ]
+      }
+    },
+    "identitypoolE2A6D099": {
+      "Type": "AWS::Cognito::IdentityPool",
+      "Properties": {
+        "AllowUnauthenticatedIdentities": false,
+        "AllowClassicFlow": true,
+        "CognitoIdentityProviders": [],
```

Comment:
@smguggen it looks like the integration tests are _not_ testing the cognito identity providers.

Reply:
@smguggen did the integration tests succeed? I tried running them and received a lot of errors.

How are you generating the `integ.identitypool.expected.json`?

Reply:
@corymhall To get the expected template I'm running `app.synth` manually to generate a new CFN template. Otherwise when the codes change on all the internally produced resource names, the expected file won't reflect it.

Reply:
Can you share some of the error messages to see if they're just around incorrect custom names or if other things failed as well?

Reply:
You should generate the expected template by running

```
yarn integ integ.identitypool.js
```

That will actually perform a deployment and make sure it is successful and then output the template in the expected.json file.

Reply:
@corymhall Well I just completely whiffed on that one, didn't I? 😂😂😂  

The reason I was so sure that worked (`yarn.integ` had always given me credentials problems) was that I was using this as an extension library in one of my side projects, and in that project I was using this feature. Well, turns out I'd forgotten about a bug I fixed here weeks ago that was causing `IdentityPool.cognitoIdentityProviders` to always be an empty array, so the Identity Pool had only been using an OIDC provider directly and wasn't using the user pool at all.

Anyway lessons learned. Thanks for being so thorough about that, changes have been made, so tell me what you think.